lib/cfme/cloud_services/data_collector.rb

```diff
@@ -24,97 +24,33 @@ def fetch_manifest
     JSON.parse(raw_manifest)
   end
 
+  def host_for_manifest_request
+    {:scheme => "http", :host => "localhost", :port => 3000}
+  end
+
+  ACCOUNT_NUMBER = "cfme-tenant".freeze
```

Comment:
is this ok for manifest request or should I get it from cfme subscription ?
thanks!